Sources/StreamChat/Models/CurrentUser.swift

```diff
@@ -92,10 +93,12 @@ public class _CurrentChatUser<ExtraData: UserExtraData>: _ChatUser<ExtraData> {
             imageURL: imageURL,
             isOnline: isOnline,
             isBanned: isBanned,
+            isFlaggedByCurrentUser: false,
```

Comment:
Is it possible to flag yourself? 😄

Reply:
That is a question that I have but I'm assuming you can't, that would be weird 😂

Reply:
Maybe when you make a really stupid joke and you want to self-acknowledge that? 🤔